components/Order/order_states.rst

```diff
@@ -4,34 +4,29 @@ Order States
 
-+-------------------+------------------------------------------------+
-| State             | Description                                    |
-+===================+================================================+
-| cart              | Unconfirmed order, ready to add/remove items   |
-+-------------------+------------------------------------------------+
-| cart_locked       | Cart which should not be removed when expired  |
-+-------------------+------------------------------------------------+
-| pending           | Order waiting for confirmation                 |
-+-------------------+------------------------------------------------+
-| confirmed         | Confirmed and completed order                  |
-+-------------------+------------------------------------------------+
-| shipped           | Order has been shipped to the customer         |
-+-------------------+------------------------------------------------+
-| abandoned         | Been waiting too long for confirmation         |
-+-------------------+------------------------------------------------+
-| cancelled         | Cancelled by customer or manager               |
-+-------------------+------------------------------------------------+
-| returned          | Order has been returned and refunded           |
-+-------------------+------------------------------------------------+
-
-.. note::
++-------------------+-------------+-----------------------------------------------+
+| Related constant  | State       | Description                                   |
++===================+=============+===============================================+
+| STATE_CART        | cart        | Unconfirmed order, ready to add/remove items  |
++-------------------+-------------+-----------------------------------------------+
+| STATE_CART_LOCKED | cart_locked | Cart which should not be removed when expired |
++-------------------+-------------+-----------------------------------------------+
+| STATE_PENDING     | pending     | Order waiting for confirmation                |
++-------------------+-------------+-----------------------------------------------+
+| STATE_CONFIRMED   | confirmed   | Confirmed and completed order                 |
++-------------------+-------------+-----------------------------------------------+
+| STATE_SHIPPED     | shipped     | Order has been shipped to the customer        |
++-------------------+-------------+-----------------------------------------------+
+| STATE_ABANDONED   | abandoned   | Been waiting too long for confirmation        |
++-------------------+-------------+-----------------------------------------------+
+| STATE_CANCELLED   | cancelled   | Cancelled by customer or manager              |
++-------------------+-------------+-----------------------------------------------+
+| STATE_RETURNED    | returned    | Order has been returned and refunded          |
++-------------------+-------------+-----------------------------------------------+
```

Comment:
:+1: We should follow this convention for all other states. In future, I'd like to add some graphs (imgs) with transitions etc. but this is good for now.